### ISSUE TITLE
`/**` in @param description causes false errors

### ISSUE THREAD

**Issue by jaydenseric** [gajus/eslint-plugin-jsdoc#568]
Using  `/**` in a JSDoc `@param` description for a function causes false errors, that when autofixed, deletes most of the existing JSDoc.

I actually need to be able to write inline code containing this because I'm documenting code to do with authoring JSDoc. It's expected that a multiline comment end (`*/`) needs to be escaped (`*\/`) to prevent the JSDoc comment block from closing early, but it is unexpected to have to also escape comment starts (`/**` -> `/\**`).

## Expected behavior

With this function and the `jsdoc/require-param` rule enabled:

```js
/**
 * Description.
 * @param {string} b Description `/**`.
 */
module.exports = function a(b) {
  console.info(b);
};
```

There should be no errors.

## Actual behavior

There is a `jsdoc/require-param` error:

```
Missing JSDoc @param "b" declaration.
```

It autofixes to this:

```js
/**
 * `.
 * @param b
 */
module.exports = function a(b) {
  console.info(b);
};
```

## ESLint Config

```json
{
  "rules": {
    "jsdoc/require-param": "error"
  }
}
```

## ESLint sample

See above reproduction steps.

## Environment

* Node version: v14.3.0
* ESLint version v7.1.0
* `eslint-plugin-jsdoc` version: v27.0.4

**Comment by brettz9** [comment]
This is an issue in underlying `comment-parser`. Please follow at https://github.com/syavorsky/comment-parser/pull/89 . Closing here, as we tend to update fairly frequently (once that may be fixed).
